pentaho-database-gwt/src/org/pentaho/ui/database/event/DataHandler.java

```diff
@@ -1131,11 +1131,28 @@ public void restoreDefaults() {
         }
         item.getRow().addCellText( 2, defaultValue );
         item.getRow().addCellText( 0, "false" );
+        if ( "validationQuery".equals( parameterName ) ) {
+          item.getRow().setParentTreeItem( item );
+          item.getRow().getCell( 0 ).setTreeRowParent( findRow( "testOnBorrow" ) );
```

Comment:
findRow can return null if there is no parameter which we try find. I think its ok.